lib/Search/Provider.php

```diff
@@ -103,7 +103,9 @@ public function search(IUser $user, ISearchQuery $query): SearchResult {
 				}
 
 				return new SearchResultEntry(
-					'',
+					$this->urlGenerator->linkToRoute('mail.avatars.image', [
```

Comment:
since it's an "avatar" I think it should be rounded, so set the last parameter of the `SearchResultEntry` to true